modules/core/src/lib/tooltip.ts

```diff
@@ -41,23 +45,48 @@ export type TooltipContent =
       style?: Partial<CSSStyleDeclaration>;
     };
 
-export default class Tooltip {
-  private el: HTMLDivElement | null = null;
-
+export default class Tooltip implements Widget {
   isVisible: boolean = false;
+  deck?: Deck;
+  element?: HTMLDivElement;
+  lastViewport?: Viewport;
+
+  onAdd({deck}: {deck: Deck}) {
```

Comment:
I guess one doesn't have to type out the return type when implementing an interface but I guess somehow the omission stands out to me.